backend/functions/lib/adapters/gateways/dxtr/orders.dxtr.gateway.ts

```diff
@@ -66,7 +67,11 @@ export class DxtrOrdersGateway implements OrdersGateway {
 
       orderSync.orders = rawOrders
         .map((rawOrder) => {
-          if (orderSync.maxTxId < rawOrder.txId) orderSync.maxTxId = rawOrder.txId;
+          const txIdAsInt = parseInt(rawOrder.txId);
+          if (maxTxIdasInt < txIdAsInt) {
```

Comment:
I'm not a fan of variable names like this?  If it takes me a minute to figure out what that thing is, it's not named properly.

Reply:
Yeah, I have a note to refactor this code too. Can rename them as part of that.

Reply:
consistency of camelCase is also not present